Makefile

```diff
@@ -21,6 +21,14 @@ help: # Display help
 			printf "\033[36m%-30s\033[0m %s\n", $$1, $$NF \
 		}' $(MAKEFILE_LIST) | sort
 
+.PHONY: format
+format: ## Autoformatting for codebase.
+	shfmt -f ./ | xargs shfmt -w
+
+.PHONY: lint
+lint: ## Lint checking for codebase.
+	find . -type f -name "*.sh" -print0 | xargs -0 shellcheck
+
```

Comment:
I am not against using xarg at all, so just out of curiosity, with not using globs? like `shfmt -f */**.sh` and `shellcheck */**.sh`?

Reply:
That globbing won't act in the same way as if there was a recursive search for the files. To show this, try modifying `examples/buildpacks/buildpacks.sh` and running `shfmt` or `shellcheck` with your globbing syntax. It'll only catch one directory down rather than recursing. 

I'm surprised for either there's not a `-recursive` option like `terraform fmt`, but alas!

Reply:
Ah ok, it is because I made a mistake in my glob 🤦  If you use `**/*.sh` it works. But anyway, `xargs` is good too 👍